portality/lib/formulaic.py

```diff
@@ -100,6 +100,8 @@
 from copy import deepcopy
 from wtforms import Form
 from wtforms.fields.core import UnboundField, FieldList, FormField
+
+from portality import datasets
```

Comment:
This is no longer in use, so can be removed

Reply:
done